Infrastructure/Helper/DbSchemaCheck.cs

```diff
@@ -73,5 +74,40 @@ private static void EnsurePaidCashColumnExistsInSalesTable(SQLiteConnection conn
                 }
             }
         }
+
+        private static void EnsureStorageAddressColumnDeletedFromAvailabilityTable(SQLiteConnection connection)
+        {
+            using (var transaction = connection.BeginTransaction())
+            {
+                using (var cmd = new SQLiteCommand("", connection, transaction))
+                {
+                    cmd.CommandText = $"SELECT COUNT(*) AS ColumnExists FROM sqlite_master WHERE type = 'table' AND name = 'Avaliability' AND sql LIKE '%StorageAdress%';";
+                    try
+                    {
+                        bool columnExists = Convert.ToInt32(cmd.ExecuteScalar()) != 0;
+                        if (columnExists)
+                        {
+                            cmd.Parameters.Clear();
+                            cmd.CommandText = $"CREATE TABLE Avaliability_backup " +
```

Comment:
Why do you need to recreate the table? Can't you just remove the column, what's the problem?

Reply:
'drop column' query doesn't work obviously in SQLite; 
removed any StorageAddress mentions from DB operations and from Purchase UI, added 2nd goal in User Story #45 (drop SA column from DB table without recreating Availability table)